### PR TITLE
Cluster Settings: Updated Channel helper text and links

### PR DESCRIPTION
**Pull request by dtaylor113**
https://issues.redhat.com/browse/CONSOLE-2268

**Updated Subscription link and added Popover to Channel title**
image

**...the Popover**
image

### CODE REVIEW

File: frontend/public/components/cluster-settings/cluster-settings.tsx [hunk 1]
```diff
@@ -222,6 +222,31 @@ export const CurrentVersionHeader: React.SFC<CurrentVersionProps> = ({ cv }) =>
   );
 };
 
+const ChannelHeader: React.FC<{}> = () => {
+  return (
+    <Popover
+      headerContent={<>Channel</>}
+      bodyContent={
+        <div>
```
Comment (rhamilto) [inline frontend/public/components/cluster-settings/cluster-settings.tsx:230]: Nit:  use `<>` since you don't need an HTML element.

Comment (dtaylor113) [reply frontend/public/components/cluster-settings/cluster-settings.tsx]: ok, fixed

**Comment by dtaylor113** [comment]
> Small wording detail

@megan-hall, ok 'updated' :-)

image

**Pull request merged**